src/ozmaps/mapmanager.c

```diff
@@ -248,7 +248,8 @@ SDL_Surface *getmap(int mapflags,      /* instructions on which map to get */
       camera->y = 0;
       return (mapNow);
     }
-  }
+  } else 
+     return (mapNow);  /* should never reach here */
 }
```

Comment:
You didn't run `clang-format -i` after do the modifications.

Reply:
Yeah, you should really fix the format in the commit that introduced the problem, not create a new commit to fix the format.